docs/feature-flag/feature_flag

```diff
@@ -0,0 +1,27 @@
+# How to Add a Feature Flag on Firebase
+
+## 1. Go to Remote Config on Firebase
```

Comment:
[suggestion]: add the screenshots to each step

Reply:
Added.